src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -1,659 +1,91 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit: There is a new sub directory workers\java under WebHost. Can you fix the capitalization of the paths to follow all the others? Also, the wwwroot directory that is part of the project seems to be missing - remove from project?

Reply:
The `workers/java` and `workers/node` directories come from the language worker nugets - they are copied as content to the output directory.
I want to keep lowercase paths just for better consistency across platforms.

Reply:
The `workers` come from the packages, @mamaso , not sure those would need to be content on the project.

Removed `wwwroot`

Reply:
Currently they are expected to be found in the `bin/workers` directory - easiest way I've found to manage that is by treating them as content. I might be able to set a property that hides the content from VS

Reply:
You can also have a custom target in the package that moves the files over.